src/main/resources/metafile-schema.json

```diff
@@ -192,7 +220,11 @@
             "type": "string",
             "title": "contextual information describing this item"
           }
-        }
+        },
+        "required": [
+          "document_control_number",
+          "item_type"
```

Comment:
not sure if item type is mandatory ?

Reply:
item type is added in required fields list in the confluence. we can discuss later